Modules/Filtering/Smoothing/wrapping/test/MedianImageFilterTest.py

```diff
@@ -25,11 +25,8 @@
 from sys import argv, version_info
 import warnings
 
-if version_info >= (3, 8):
-    from typing import Sequence, TypeVar, get_type_hints, get_args, get_origin, Union
-else:
-    from typing import Sequence, TypeVar, Union
-
+from typing import TypeVar, get_type_hints, get_args, get_origin, Union
```

Comment:
Does keeping the `Sequence` import from `typing` instead of `collections.abc` address the failure?

Reply:
@thewtex That would be one way, but Sequence is deprecated in typing in 3.9 and the recommended import is from collections.abc.

I've fixed this by importing from collections.abc.  See the new patchset.